src/main/java/org/sonarsource/sonarlint/ls/AnalysisClientInputFile.java

```diff
@@ -107,6 +107,7 @@ private static Language toSqLanguage(@Nullable String clientLanguageId) {
         // See https://github.com/forcedotcom/salesforcedx-vscode/blob/5e4b7715d1cb3d1ee2780780ed63f70f58e93b20/packages/salesforcedx-vscode-apex/package.json#L273
         Language.APEX;
       case "yaml" -> Language.YAML;
+      case "json" -> Language.JSON;
```

Comment:
Why is this needed?

Reply:
I assumed that it should be "forced" like for YAML, but apparently, both this line and the line above are not necessary. Good catch!